core/src/main/java/io/github/tcdl/api/ResponderServer.java

```diff
@@ -55,7 +55,7 @@ public ResponderServer listen() {
         channelManager.subscribe(namespace,
                 message -> {
                         LOG.debug("Received message with id {} from topic {}", message.getId(), namespace);
-                        Responder responder = new Responder(messageTemplate, message, msbContext);
+                        ResponderImpl responder = new ResponderImpl(messageTemplate, message, msbContext);
```

Comment:
this is internal logic - private methods, and also we make use of ResponderImpl.getOriginalMessage which is not part of Responder contract